inventory/src/AddProduct.java

```diff
@@ -1,6 +1,12 @@
 public class AddProduct extends Modules implements ModuleSpecific{
-    @Override
+
+
     public void driver() {
+        System.out.println("This is the driver");
```

Comment:
This doesn't seem like how it should work?

Reply:
You're right. This was a duplicate method and was removed in the next request.